db/migrate/20200303143426_backfill_removals_removed_at.rb

```diff
@@ -0,0 +1,13 @@
+class BackfillRemovalsRemovedAt < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+
+  class Removal < ApplicationRecord
+    has_one :status, as: :details
+  end
+
+  def up
+    Removal.all.each { |removal| removal.update!(removed_at: removal.created_at) }
```

Comment:
Super tiny comment, you can use `find_each` here instead of `all.each` if you so wish which reads a little nicer!

Reply:
It's also more performant to do find_each as it pulls out in queries of 1000 - though not too big an issue as we probably have 5.

You can do `Removal.where(removed_at: nil)` so you're only replacing items that aren't already set (though I guess it's unlikely there are any given we have to do the removals)

Reply:
Yeah I didn't bother with find_each since there are only 6 records.